docs/sources/datasources/influxdb/_index.md

```diff
@@ -216,7 +215,7 @@ datasources:
 apiVersion: 1
 
 datasources:
-  - name: InfluxDB_v2_InfluxQL
+  - name: InfluxDB_v3_InfluxQL
```

Comment:
In this example `jsonData` must have `version: SQL`
https://github.com/grafana/grafana/pull/84119/files/57e2da3321179e9041f52ddcff1f953c982cf528#r1518798191

Reply:
I've updated the branch to reflect this @itsmylife